### PR TITLE
Hotfix user permissions

### PR DESCRIPTION
**Pull request by aganpule**
- added step defs the different user types
- Fixed small bug in the events controller & events index view
- Updated events + resources_permissions feature

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -13,9 +13,9 @@ def authenticate_event_admin!
   # GET /events
   # GET /events.json
   def index
-    @core_member = false
-    if !current_user.nil? and current_user.is_core_member
-      @core_member = true
+    @event_admin = false
```
Comment (susannasouv) [inline app/controllers/events_controller.rb:16]: Talked with @aganpule offline about this. Will cause some controller tests to fail. fixing

Comment (susannasouv) [reply app/controllers/events_controller.rb:16]: Addressed in other PR#85

**Review by susannasouv — APPROVED** [review]
lgtm

**Review by LizzySteger — APPROVED** [review]
Looks good

**Pull request merged**